### PR TITLE
Korean translation work

### PR DESCRIPTION
**Pull request by junbeomlee**
fixed: #253 
We are in the process of translating the documents in the ibc folder. 
Work in progress can be found here(https://github.com/DE-labtory/ics/pulls)

Please check the finished translation.
 - RESEARCH.md
 - ECOSYSTEM.md
 - 1_IBC_TERMINOLOGY.md

It would be nice if you could tell me the contribution guide!!

### CODE REVIEW

**Comment by cwgoes** [comment]
Unfortunately I can't review the Korean translation - can you @mossid?

A few notes:
- I don't think you meant to add `.DS_Store`
- Make sure to note that the translation is up-to-date with the spec as of a certain commit / date
- Give yourself credit for the translation!

**Comment by junbeomlee** [comment]
plz @mossid 
We are also working on translation of the ics spec and We've done about 5 of them now.
After internal review, we will send you an additional PR and We will remove the `.DS_Store` file.

File: ko/ibc/1_IBC_TERMINOLOGY.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+*중계 프로세스* 는 블록체인의 상태를 스캔하고 트랜잭션을 제출하여 둘 이상의 블록체인간에 IBC 패킷 데이터 및 메타 데이터를 전달하는 오프 체인 프로세스입니다.
+
+### 상태 머신
+
+특정 체인의 *상태 머신* 은 상태의 구조 뿐만 아니라 체인의 합의 알고리즘에 의해 합의 된 현재 상태를 기반으로 상태 전환(state-transitions)을 유발하는 유효한 트랜잭션을 결정하는 규칙 세트도 정의합니다.
+
+### 합의
+
+*합의* 알고리즘은 분산 원장을 운영하는 일련의 프로세스가 일반적으로 제한된 수의 비잔틴 결함이있는 상태에서도 동일한 상태로 합의하기 위해 사용되는 프로토콜입니다.
```
Comment (mossid) [inline ko/ibc/1_IBC_TERMINOLOGY.md]:
```suggestion
*합의* 알고리즘은 분산 원장을 운영하는 일련의 프로세스들이, 비잔틴 결함이 주어진 상한선 미만으로 있는 상태에서는, 동일한 상태로 합의할 수 있도록 사용하는 프로토콜입니다.
```

File: ko/ibc/1_IBC_TERMINOLOGY.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+### 합의 상태
+
+*합의 상태* 는 해당 합의 알고리즘의 결과에 관하여 증명하는데 필요한 합의 알고리즘의 상태에 대한 정보 집합입니다.
```
Comment (mossid) [inline ko/ibc/1_IBC_TERMINOLOGY.md]:
```suggestion
*합의 상태* 는 합의 알고리즘의 상태에 대한 정보 집합으로, 해당 합의 알고리즘의 결과에 대한 증명을 검증하는데 필요합니다(e.g. 서명된 헤더의 커밋 루트)
```

File: ko/ibc/1_IBC_TERMINOLOGY.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+### Commitment
+
+*A cryptographic commitment* 는 매핑에서 키/값 쌍의 멤버 자격 또는 비 멤버쉽을 저렴하게 확인하는 방법으로, 짧은 감시 문자열로 매핑을 커밋 할 수 있습니다.
```
Comment (mossid) [inline ko/ibc/1_IBC_TERMINOLOGY.md]:
```suggestion
*암호학적 커밋* 은 매핑에서 키/값 쌍의 포함 또는 비포함 여부를 저렴하게 확인하는 방법입니다. 여기서 매핑은 짧은 증명 문자열로 커밋될 수 있습니다.
```

File: ko/ibc/1_IBC_TERMINOLOGY.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+### 헤더
+
+*헤더* 는 "가벼운 클라이언트" 알고리즘에 의해 잘 정의 된 방식으로 검증 될 수 있는 현재 상태에 대한 약속을 포함하여 특정 블록 체인의 합의 상태에 대한 업데이트입니다.
```
Comment (mossid) [inline ko/ibc/1_IBC_TERMINOLOGY.md]:
```suggestion
*헤더* 는 "라이트 클라이언트" 알고리즘에 의해 잘 정의 된 방식으로 검증 될 수 있는 현재 상태에 대한 커밋을 포함하여 특정 블록 체인의 합의 상태에 대한 업데이트입니다.
```

File: ko/ibc/1_IBC_TERMINOLOGY.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+### CommitmentProof
+
+*CommitmentProof* 은 특정 키가 기록 대상 세트의 특정 값에 맵핑되는지 여부를 증명하는 증명 구조입니다.
```
Comment (mossid) [inline ko/ibc/1_IBC_TERMINOLOGY.md]:
```suggestion
*커밋 증명* 은 특정 키가 커밋된 집합 안에서 특정 밸류로 대응되는지 또는 대응되는 밸류가 없는지를 증명하는 증명 자료구조입니다.
```

File: ko/ibc/1_IBC_TERMINOLOGY.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+
+### 핸들러 모듈
+
+[ICS 25](../spec/ics-025-handler-module)를 구현한 IBC *핸들러 모듈* 은 클라이언트, 커넥션 및 채널을 관리하고, 증명을 확인하고, 패킷에 대한 적절한 기록을 저장하는 상태 머신 내의 {em1}모듈{/em1} 입니다.
```
Comment (mossid) [inline ko/ibc/1_IBC_TERMINOLOGY.md]:
```suggestion
[ICS 25](../spec/ics-025-handler-module)를 구현한 IBC *핸들러 모듈* 은 클라이언트, 커넥션 및 채널을 관리하고, 증명을 검증하고, 패킷에 대한 적절한 커밋을 저장하는 상태 머신 내의 {em1}모듈{/em1} 입니다.
```

File: ko/ibc/1_IBC_TERMINOLOGY.md [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+
+### 라우팅 모듈
+
+[ICS 26](../spec/ics-026-routing-module)를 구현한 IBC *라우팅 모듈* 은 상태 머신 내의 모듈로서, 라우팅 모듈의 외부 인터페이스를 이용하는 호스트 상태 머신상의 핸들러 모듈과 다른 모듈 사이에서 패킷을 라우팅 합니다.
+
+### 데이터 그램
+
+*데이터 그램* 은 일부 물리적 네트워크를 통해 전송되는 바이트 스트링이며 원장의 상태 시스템에서 구현 된 IBC 라우팅 모듈에 의해 처리됩니다. 일부 구현들에서, 데이터 그램은 다른 정보 (예를 들어, 스팸 방지 비용, 재생 공격(replay attack) 방지를 위한 논스, IBC 핸들러로 라우팅하기위한 타입 식별자 등)를 포함하는 특정 원장의 트랜잭션 또는 메시지 데이터 구조의 필드 일 수 있습니다. 모든 IBC의 하위 프로토콜 (예: 커넥션(Connection) 열기, 채널 만들기, 패킷 보내기)은 라우팅 모듈을 통해 들어온 데이터를 처리하기 위한 데이터 그램 및 프로토콜 세트로 정의됩니다.
```
Comment (mossid) [inline ko/ibc/1_IBC_TERMINOLOGY.md]:
```suggestion
*데이터 그램* 은 물리적 네트워크를 통해 전송되는 바이트 스트링이며, 원장의 상태 머신에서 구현 된 IBC 라우팅 모듈에 의해 처리됩니다. 일부 구현들에서, 데이터 그램은 다른 정보 (예를 들어, 스팸 방지 비용, 재생 공격(replay attack) 방지를 위한 논스, IBC 핸들러로 라우팅하기위한 타입 식별자 등)를 포함하는 특정 원장의 트랜잭션 또는 메시지 데이터 구조의 필드 일 수 있습니다. 모든 IBC의 하위 프로토콜 (예: 커넥션(Connection) 열기, 채널 만들기, 패킷 보내기)은 라우팅 모듈을 통해 들어온 데이터를 처리하기 위한 데이터 그램 및 프로토콜 세트로 정의됩니다.
```

File: ko/ibc/1_IBC_TERMINOLOGY.md [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+
+### 커넥션(Connection)
+
+*커넥션(Connection)* 은 다른 원장의 합의 상태에 대한 정보를 포함하는 두 체인의 영구 데이터 구조의 집합입니다. 한 체인의 합의 상태를 업데이트하면 다른 체인의 커넥션(Connection) 상태가 변경됩니다.
+
+### 채널
+
+*채널* 은 패킷 순서 지정, exactly-once delivery 및 재생 방지를 용이하게하기 위해 메타 데이터를 포함하는 두 체인의 영구 데이터 구조 세트입니다. 채널을 통해 전송 된 패킷은 내부 상태를 변경합니다. 채널은 다-대-일 관계의 커넥션(Connection)과 연관되어 있습니다. 단일 커넥션은 여러 개의 연관된 채널을 가질 수 있으며 모든 채널은 하나의 연관된 커넥션(Connection)을 가져야하며 이는 채널을 생성하기 전에 생성해야합니다.
```
Comment (mossid) [inline ko/ibc/1_IBC_TERMINOLOGY.md]:
```suggestion
*채널* 은 패킷 순서 지정, exactly-once delivery 및 이중 실행 방지를 용이하게하기 위해 메타 데이터를 포함하는 두 체인의 영구 데이터 구조 세트입니다. 채널을 통해 전송 된 패킷은 내부 상태를 변경합니다. 채널은 커넥션(Connection)과 다대일 관계로 연관되어 있습니다. 한 커넥션은 여러 개의 연관된 채널을 가질 수 있으며 모든 채널은 하나의 연관된 커넥션(Connection)을 가져야하며, 커넥션은 그와 연관된 채널들에 앞서 생성되어 있어야 합니다.
```

File: ko/ibc/1_IBC_TERMINOLOGY.md [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+
+### 패킷
+
+*패킷* 은 순서 관련 메타 데이터 (IBC 사양에 의해 정의 됨) 및 패킷 *데이터* 라고 하는 오파크(opaque) 필드(응용 프로그램 계층에 의해 정의 된 의미, 예를 들어 토큰 양 및 명칭)를 갖는 특정 데이터 구조입니다. 패킷은 특정 채널(좀 더 확장하여 설명하면 특정 커넥션)을 통해 전송됩니다.
```
Comment (mossid) [inline ko/ibc/1_IBC_TERMINOLOGY.md]:
```suggestion
*패킷* 은 순서 관련 메타 데이터 (IBC 스펙에 의해 정의 됨) 및 패킷 *데이터* (어플리케이션 계층에서 의미가 정의됨, 예를 들어 토큰 개수 및 이름) 라고 하는 오파크(opaque) 필드를 갖는 특정 데이터 구조입니다. 패킷은 특정 채널(좀 더 확장하여 설명하면 특정 커넥션)을 통해 전송됩니다.
```

File: ko/ibc/1_IBC_TERMINOLOGY.md [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+
+### 모듈
+
+*모듈* 은 특정 블록 체인의 상태 머신의 하위 구성 요소로, IBC 핸들러와 상호 작용하고 전송하거나 받는 특정 IBC 패킷의 *데이터* 필드에 따라 상태를 변경 시킬 수 있습니다 (예: 토큰 발행 혹은 소각).
+
+### 핸드셰이크(Handshake)
+
+*핸드 셰이크* 는 여러 데이터 그램(Datagram)을 포함하는 특정 하위 프로토콜로, 일반적으로 두 개의 관련된 체인에서 서로의 합의 알고리즘에 대해 신뢰 할 수 있는 상태와 같은 일부 공통 상태를 초기화하는 데 사용됩니다.
```
Comment (mossid) [inline ko/ibc/1_IBC_TERMINOLOGY.md]:
```suggestion
*핸드 셰이크* 는 여러 데이터그램(Datagram)을 포함하는 하위 프로토콜들의 특정한 묶음으로, 일반적으로 두 개의 관련된 체인에서 서로의 합의 알고리즘에 대해 신뢰 할 수 있는 상태와 같은 일부 공통 상태를 초기화하는 데 사용됩니다.
```

File: ko/ibc/1_IBC_TERMINOLOGY.md [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+
+### 하위 프로토콜
+
+하위 프로토콜은 블록 체인을 구현하는 IBC 핸들러 모듈에 의해 구현되어야하는 데이터 그램 종류 및 함수 세트로 정의됩니다.
```
Comment (mossid) [inline ko/ibc/1_IBC_TERMINOLOGY.md]:
```suggestion
하위 프로토콜은 IBC를 구현하는 블록체인에서 핸들러 모듈의 형태로 구현되어야하는 데이터그램 종류 및 함수 집합으로 정의됩니다.
```

File: ko/ibc/1_IBC_TERMINOLOGY.md [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+
+### 인증
+
+*인증* 은 실제로 데이터 그램이 IBC 핸들러에 의해 정의 된 방식으로 특정 체인에 의해 전송되도록하는 속성입니다.
+
+## 속성 정의
+
+### 완결성
+
+*완결성* 은 validator 집합의 동작에 대한 특정 가정에 따라 특정 블록이 되돌리지 않을 것이라는 합의 알고리즘에 의해 제공되는 수량화 가능한 보증입니다. IBC 프로토콜은 절대적일 필요는 없지만 완결성을 요구합니다. (예를 들어, Nakamoto 합의 알고리즘의 임계값 완결성 장치는 채굴자의 행동 방식에 대한 경제적인 가정에 따라 완결성을 제공합니다.)
```
Comment (mossid) [inline ko/ibc/1_IBC_TERMINOLOGY.md]:
```suggestion
*완결성* 은 특정한 블록이 되돌려지지 않을 것이라고 합의 알고리즘이 확률적으로 제공하는 보증이며, 검증인 집합의 행동에 주어진 특정한 가정에 따릅니다. IBC 프로토콜은 완결성을 요구하지만 완전히 완결적이여야 할 필요는 없습니다. (예를 들어, Nakamoto 합의 알고리즘의 threshold finality gadget은 채굴자의 행동 방식에 대한 경제적인 가정에 따라 주관적인 완결성을 제공합니다.)
```

File: ko/ibc/1_IBC_TERMINOLOGY.md [hunk 13]
```diff
@@ -0,0 +1,155 @@
...
+
+### 오작동
+
+*오작동* 은 합의 알고리즘에 의해 정의되고 해당 합의 알고리즘의 라이트 클라이언트가 감지 할 수있는 합의 오류의 유형 입니다.
+
+### Equivocation
+
+*Equivocation* 은 유효하지 않은 방식으로 여러 개의 다른 후속 작업에 대한 투표를 단일 블록에 서명하는 validator가 저지른 특정 합의 오류의 유형입니다.
```
Comment (mossid) [inline ko/ibc/1_IBC_TERMINOLOGY.md]:
```suggestion
*Equivocation* 은 합의 오류의 특정한 유형 중 하나로, 하나 또는 여러 검증인이 서로 다른 여러 자식 블록에 서명하는 유효하지 않은 행위입니다. 
```

File: ko/ibc/1_IBC_TERMINOLOGY.md [hunk 14]
```diff
@@ -0,0 +1,155 @@
...
+*오작동* 은 합의 알고리즘에 의해 정의되고 해당 합의 알고리즘의 라이트 클라이언트가 감지 할 수있는 합의 오류의 유형 입니다.
+
+### Equivocation
+
+*Equivocation* 은 유효하지 않은 방식으로 여러 개의 다른 후속 작업에 대한 투표를 단일 블록에 서명하는 validator가 저지른 특정 합의 오류의 유형입니다. 
+모든 Equivocation은 오작동 입니다.
+
+### 데이터 가용성
+
+*데이터 가용성* 은 일정 기간 내에 machine의 상태에서 데이터를 조회할 수 있는 오프 체인 replayer 프로세스 기능입니다.
```
Comment (mossid) [inline ko/ibc/1_IBC_TERMINOLOGY.md]:
```suggestion
*데이터 가용성* 은 주어진 시간 제한 내에 머신의 상태에서 오프체인 relayer 프로세스가 데이터를 조회할 수 있는 속성입니다.
```

**Comment by cwgoes** [comment]
Bump - have all requested changes been addressed? cc @mossid

**Comment by cwgoes** [comment]
cc @dogemos if you are feeling inspired

**Comment by cwgoes** [comment]
Please move the files to `translation/ko`.

**Comment by cwgoes** [comment]
Also I don't think you meant to commit `.DS_Store`.

**Comment by junbeomlee** [comment]
sure! i will fix it

**Comment by dogemos** [comment]
It appears that this PR has duplicated documents that have been merged with changes (https://github.com/cosmos/ics/pull/295) and are in the process of making changes (https://github.com/cosmos/ics/pull/298).

@junbeomlee Could we close this PR and create a new PR for the rest of the documents (the ones that don't have its own PRs):
- `2_IBC_ARCHITECTURE.md`
- `3_IBC_DESIGN_PRINCIPLES.md`
- `4_IBC_USECASES.md`
- `5_IBC_DESIGN_PATTERNS.md`
- `README.md`

**Pull request closed without merging**